cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -257,9 +258,63 @@ class SpatialTest extends DocumentingTest {
           |<<cypher-spatial-crs, four coordinate reference systems>>.
           |This allows for both <<schema-index-use-a-single-property-index-with-where-using-equality, equality>>
           |and <<schema-index-use-index-with-where-using-range-comparisons, range>> queries using exactly the same syntax and behaviour as for other property types.
+          |If two range predicates are used, setting minimum and maximum points, this will effectively result in a
+          |<<schema-index-use-index-when-executing-a-spatial-bounding-box-search, 'bounding box query'>>.
           |In addition, queries using the `distance` function can, under the right conditions, also use the index, as described in the section
-          |<<schema-index-use-index-when-executing-a-spatial-distance-search, 'Use index when executing a spatial distance search'>>
+          |<<schema-index-use-index-when-executing-a-spatial-distance-search, 'Use index when executing a spatial distance search'>>.
         """.stripMargin)
     }
+    section("Comparability and Orderability", "cypher-comparability-orderability") {
+      p(
+        """
+          |Points with different CRS are not comparable. This means that any function operating on two points of different type will return null.
+          |This is true of the distance function as well as inequality comparisons. If these are used in predicate, they will cause the associate match
+          |to return no results.
+        """.stripMargin
+      )
+      query(
+        """WITH point({x:3, y:0}) AS p2d, point({x:0, y:4, z:1}) as p3d
+          |RETURN distance(p2d,p3d), p2d < p3d, p2d = p3d, p2d <> p3d, distance(p2d,point({x:p3d.x, y:p3d.y}))""".stripMargin, ResultAssertions((r) => {
+          r.hasNext should be(true)
+          val record = r.next()
+          withClue("Expect the invalid distance function to return null") {
+            (record("distance(p2d,p3d)") == null) should be(true)
+          }
+          withClue("Expect the inequality test to return null") {
+            (record("p2d < p3d") == null) should be(true)
+          }
+          withClue("Expect the equality test to return false") {
+            (record("p2d = p3d") == false) should be(true)
+          }
+          withClue("Expect the negative equality test to return true") {
+            (record("p2d <> p3d") == true) should be(true)
+          }
+        })) {
+        resultTable()
+      }
+      p(
+        """
+          |However, all types are orderable. The Point types will be ordered after Numbers and before Temporal types.
```

Comment:
New sentence, new line

Reply:
Hyperlink "Point"

Reply:
I could not find something to link it to. It is a type, and the section on types simply lists Point along with all other types. Or perhaps you want a link to the `point` function? In that case should the section on types also link to the `point` function?